docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+processor:
+  build: ../data-processor/.
+  links:
+    - queue
+    - db
+  environment:
+    VIP_DP_AWS_ACCESS_KEY:
+    VIP_DP_AWS_SECRET_KEY:
+    VIP_DP_S3_BUCKET:
+    VIP_DP_SQS_REGION:
+    VIP_DP_SQS_QUEUE:
+    VIP_DP_SQS_FAIL_QUEUE:
+    VIP_DP_RABBITMQ_EXCHANGE:
+queue:
```

Comment:
For that I just mean the Docker link env vars will have to change when you change the name of an entry in `docker-compose.yml`. So `QUEUE_PORT_5672_TCP_ADDR` would have to become `RABBITMQ_PORT_5672_TCP_ADDR`.